src/containers/VerletLists.h

```diff
@@ -1,23 +1,156 @@
-/*
- * VerletLists.h
- *
- *  Created on: 17 Jan 2018
- *      Author: tchipevn
+/**
+ * @file VerletLists.h
+ * @author seckler
+ * @date 19.04.18
  */
 
-#ifndef SRC_CONTAINERS_VERLETLISTS_H_
-#define SRC_CONTAINERS_VERLETLISTS_H_
+#pragma once
 
-#include "ParticleContainer.h"
+#include "LinkedCells.h"
 
 namespace autopas {
 
+/**
+ * Verlet Lists container.
+ * This class builds neighbour lists for the particle interactions.
+ * This class does NOT work with RMM cells and is not intended to!
```

Comment:
Hmm I thought about this, but you need a lot of additional things for that and it also does not make too much sense:
RMM: reduced memory mode
verlet-lists: extra verlet lists to do everything

and it would complicate the implementation a lot. As we probably have to copy the data from the cells to some extra SoA to make it computationally efficient.

But of course we can look into it. But it's probably worth a paper ;)